README.md

```diff
@@ -9,9 +9,10 @@ Libocpp provides a complete implementation of OCPP 1.6. The implementation of OC
 ## Table of contents
 - [C++ implementation of OCPP](#c-implementation-of-ocpp)
   - [Table of contents](#table-of-contents)
-  - [Feature Support](#feature-support)
+  - [OCPP 1.6 Support](#feature-support)
```

Comment:
nit: Looking at it now we could just omit the third level but this is just personal preference.